### DISCUSSION TITLE
Any unexpected issues when Updating from 1.32.2 to latest version 1.37.1?

### DISCUSSION THREAD

**Discussion by Anjerlaan** [Koenkk/zigbee2mqtt#22835] [General]
Hi,

What problems can I encounter if I update from 1.32.2 to 1.37.1 on Raspbian Buster on a Rpi 3b+?
For instance Node.js version (currently I have v16.18.1)

**Comment by francisp2** [comment]
Current installation instructions: 

 Verify that the correct nodejs and npm (automatically installed with nodejs)
 version has been installed
node --version  # Should output V18.x, V20.x, V21.X
npm --version  # Should output 9.X or 10.X

**Reply by Anjerlaan** [reply]
Thanx, So I have to update from 16 to a higher version

**Reply by lux73** [reply]
stick with nodeJS v20.x because 20 has LTS Support -> https://nodejs.org/en/about/previous-releases

**Reply by Anjerlaan** [reply]
Thanx,

is it correct that, according to the installing page of zigbee2mqtt, this is (also) the way to update?

```
# Set up Node.js repository and install Node.js + required dependencies
# NOTE 1: Older i386 hardware can work with [unofficial-builds.nodejs.org](https://unofficial-builds.nodejs.org/download/release/v20.9.0/ e.g. Version 20.9.0 should work.
# NOTE 2: For Ubuntu see tip below
# NOTE 3: Curl might have to be installed first via apt update && apt install curl
sudo curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash -
sudo apt-get install -y nodejs git make g++ gcc libsystemd-dev make
```

**Reply by lux73** [reply]
seems good to me

BUT keep in Mind that your Raspbian Buster installation is EOL since late 2021 !! -> https://en.wikipedia.org/wiki/Raspberry_Pi_OS

it could threw some Errors during upgrade since old Buster packages are offline...

please consider to Ugrade your whole Raspbian to Bookworm

**Reply by Anjerlaan** [reply]
Okay, that might be a good idea!
Thanx

**Comment by lux73** [comment]
@francisp2 

I would make the font in your post even larger ^^

**Reply by francisp2** [reply]
Sorry, just copied from the Zigbee2MQTT docs, I don't know why the font is so big.
